tutorials/vue/vue-chartjs.md

```diff
@@ -0,0 +1,26 @@
+Chart.js is a great library for visualizing data and displaying it in your projects. To use it with your Vuejs projects,
+there is an excellent wrapper called `vue-chartjs`. You need both `chart.js` and `vue-chartjs` if you wish to use this
+library in your Vuejs projects. If you do not wish to install the libraries, you can instead use it directly in the
+browser via CDN. Because `vue-chartjs` is a wrapper of `chart.js`, every chart type is available for use when working
+with these libraries. When you import one of these chart types from `vue-chartjs`, you need to extend it. You can do so
+with the `extends` property or the `mixins` property. If you choose to use the `mixins` property, you must enclose the import
```

Comment:
I'd break off the `mixins` property into a separate paragraph that includes an  example of how to use `mixins`, otherwise omit it.